jenkins/Jenkinsfile

```diff
@@ -579,7 +574,10 @@ timestamps {
 
                             sh ("ls -la package")
                             uploadFiles ("package/*", "wrench", virtualPath)
-                            uploadFiles ("package/notarized/*", "wrench", virtualPath)
+
+                            def notarized_files = findFiles (glob: "package/notarized/*")
+                            if (notarized_files.size () > 0)
+                              uploadFiles ("package/notarized/*", "wrench", virtualPath)
```

Comment:
Would we like some kind of warning here? As in, we expected to find files, not were found ⚠️ ⚠️ ? Not that it block the PR.

Reply:
@mandel-macaque Should be partially covered by https://github.com/xamarin/xamarin-macios/blob/04723c517f81a3344411d44e0750d7b57e66aadb/jenkins/Jenkinsfile#L536-L542. I can add in more warnings there (for when notarization succeeds and pkgs aren't where we think they are).